code/components/nui-core/src/CefInput.cpp

```diff
@@ -355,7 +360,7 @@ static HookFunction initFunction([] ()
 {
 	g_nuiGi->QueryMayLockCursor.Connect([](int& argPtr)
 	{
-		if (HasFocus() && !g_keepInput)
+		if (nui::HasFocus() && !g_keepInput)

```

Comment:
Is this _really_ needed to be changed rather than e.g. `using nui::HasInput` somewhere out of the namespace decl?

Reply:
I guess, it's not. I'll fix it, recheck this request on current codebase and send a commit.

Reply:
Done. It's squash-rebased and tested on local server.